fn add(n: int, m: int) -> int {
    n + m
}

// TODO: both are broken 
fn main() {
    // exit(add(1, add(2, 3)))
    // exit(1 + 2 + 3)
}
